Understanding Ownership/The Slice Type/Summer Slice/task.md

```diff
@@ -1,6 +1,6 @@
 ## Summer Slice
 
-Fix this code so that it prints a fragment of an array with summer months only.
+Fix this code so that it prints a fragment of an array with [summer months](https://en.wikipedia.org/wiki/Summer#Timing) in the Northern Hemisphere only.
```

Comment:
Fix this code so that it prints a fragment of an array containing only the [summer months](https://en.wikipedia.org/wiki/Summer#Timing) in the Northern Hemisphere.